crates/fig_os_shim/src/fs.rs

```diff
@@ -3,8 +3,14 @@ use std::fs::Permissions;
 use std::io;
 #[cfg(unix)]
 use std::os::unix::ffi::OsStrExt;
-use std::path::{Path, PathBuf};
-use std::sync::{Arc, Mutex};
+use std::path::{
```

Comment:
The changes in this diff were produced by executing `cargo +nightly fmt`.
It includes no functional modifications.